Chaincase.UI/Pages/CoinJoinPage.razor

```diff
@@ -1,51 +1,142 @@
 ﻿@page "/coinjoin"
 @using Chaincase.UI.Services
+@using WalletWasabi.CoinJoin.Client.Rounds;
+@using Chaincase.Common
+@using Chaincase.UI.Components;
+
 @inject UIStateService UiStateService
 @inject NavigationManager NavigationManager
+@inject Global Global
+@inject CoinJoinViewModel CoinJoinViewModel
+
+@inherits ReactiveComponentBase<CoinJoinViewModel>
 
 <ion-content class="ion-padding">
-    <!-- CoinListTemplate -->
-    @if (IsQueuedToCoinJoin)
+
+    @if (!ViewModel.IsSynced)
     {
-        <p style="color: red;">
-            Chaincase must be open to complete a CoinJoin.
-            We'll notify you when registration ends to open.
-        </p>
+        <ion-text color="primary">
+            <h2>Once your wallet has sync'd, you can pick coins to mix!</h2>
+        </ion-text>
     }
-
-    <p>₿ @RequiredBTC required to CoinJoin</p>
-
-
-    @if (IsQueuedToCoinJoin)
+    else
     {
-        <IonButton href="/send/sent" expand="block">EXIT COINJOIN </IonButton>
+        @if (ViewModel.IsQueuedToCoinJoin)
+        {
+            <p>
+                🎉 Inputs have been registered
+            </p>
+            <p class="danger-text">
```

Comment:
Why is this a danger condition to the user?

Reply:
It could also be re-labeled `attention-seeking-text`. But "danger" is a more familiar term in the context of CSS frameworks.

Reply:
yeah danger is a common [pattern](https://ionicframework.com/docs/api/chip#properties) but should be reserved for actually dangerous activities. perhaps warning suffices